lib/restbase.js

```diff
@@ -39,6 +40,17 @@ function RESTBase (options, req) {
         this._recursionDepth = par._recursionDepth + 1;
         this._priv = par._priv;
         this.rb_config = this._priv.options.conf;
+
+        // Copy cookies to forward on first request
+        if (this._recursionDepth === 1
+            && req.headers
+            && req.headers.cookie) {
+            this.forwardedHeaders = {
```

Comment:
As an alternative, we could keep a reference to `this._req` in both root & child restbase instances. It is already defined in [line 62](https://github.com/wikimedia/restbase/pull/272/files#diff-e8ae526e9e8804c83de905dc28fa5af1R62), but isn't copied over on child instance creation yet.

As another plus, this will makes sure that the object shape remains consistent between parent & child instances, which results in more efficient code (see https://www.mediawiki.org/wiki/Learning_JavaScript#Performance_optimization).